src/pfe/file-watcher/server/test/scripts/exec.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+# Colors for success and error messages
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+BLUE='\033[0;36m'
+RESET='\033[0m'
+
+CW_DIR=~/codewind
+WEBSERVER_FILE="$CW_DIR/src/pfe/iterative-dev/test/microclimate-test/scripts/webserver.sh"
+
+function usage {
+    me=$(basename $0)
+    cat <<EOF
+Usage: $me: [-<option letter> <options value> | -h]
+Options:
+    -t # Test type, currently support 'local' and 'kube' - Mandatory
+    -s # Test suite, currently support 'functional' - Mandatory
+    -h # Display the man page
+EOF
+}
+
+function clone () {
+    curl -H "Content-type: application/json" -d '{"projectName": "'$1'", "parentPath": "'$2'", "url": "'$3'"}' "http://localhost:9090/api/v1/projects"
+}
+
+function setup {
+    DATE_NOW=$(date +"%d-%m-%Y")
+    TIME_NOW=$(date +"%H.%M.%S")
+    BUCKET_NAME=turbine-$TEST_SUITE
+    TEST_DIR=$CW_DIR/src/pfe/file-watcher/server
+    TEST_DIR_CONTAINER=/file-watcher/server
+    TEST_OUTPUT_DIR=~/test_results/$TEST_SUITE/$DATE_NOW/$TIME_NOW
+    TEST_OUTPUT=$TEST_OUTPUT_DIR/test_output.xml
+    TEST_LOG=$TEST_OUTPUT_DIR/$TEST_SUITE-test.log
+    CODEWIND_CONTAINER_ID=$(docker ps | grep codewind-pfe-amd64 | cut -d " " -f 1)
+
+    mkdir -p $TEST_OUTPUT_DIR \
+    && docker exec -it $CODEWIND_CONTAINER_ID bash -c "cd /file-watcher/server; npm install" \
```

Comment:
We shouldn't need to do npm install twice here

Reply:
One is for dependencies, the other one is for devDependencies **only**, you change your $NODE_ENV to development and that's why you only need one npm install, but the production doesn't

Reply:
The production code install is already done by default as part of the container creation.

Reply:
No, production code install dependencies for Turbine when build the PFE image, there are no dependencies that production code can install by default for the Mocha tests, and this PR is for Mocha tests. By default production code set NODE_ENV=production, then for Mocha tests, `npm install` will only install dependencies, we need to run `npm install --only=dev` to install devDependencies.

You can pull the latest master and set NODE_ENV=production, then try `npm install` for Mocha tests, you will figure out the behaviour

Reply:
Mocha tests don't use any production dependencies. All dependencies needed for mocha tests are dev dependencies. What I am saying is we don't need to do the first `npm install` as it is redundant because the production dependencies are already installed when the container is built.

Reply:
If all your new tests only need devDependencies, I am fine remove the first `npm install`